object/user_upload.go

```diff
@@ -83,6 +83,7 @@ func UploadUsers(owner string, fileId string) (bool, error) {
 
 	newUsers := []*User{}
 	for index, line := range table {
+		line := line
```

Comment:
What's this?

Reply:
reassign the iteration variable inside the loop
see above